### ISSUE TITLE
Is it possible to implement a mock server with GSS-TSIG proto?

### ISSUE THREAD

**Issue by skunkie** [bodgit/tsig#52]

**Comment by bodgit** [comment]
No, not currently. You would need to write the code to do the server side of the connection. I've tried to so something similar for SSH but I've noticed the gokrb5 implementation is missing some bits.
